docs/samples/shards/UI/Scope/1.edn

```diff
@@ -0,0 +1,22 @@
+(defloop main-wire
+  (GFX.MainWindow
+   :Contents
+   (->
+    (Setup
+     (GFX.DrawQueue) >= .ui-draw-queue
+     (GFX.UIPass .ui-draw-queue) >> .render-steps)
+    .ui-draw-queue (GFX.ClearQueue)
+
+    (UI
+     .ui-draw-queue
+     (UI.CentralPanel
+      (UI.Scope
```

Comment:
For the moment none. But it will once I implement the `Style` shard. A scope creates a child UI which can have its style overridden without affecting the rest of the UI (parents or siblings).